src/config.ts

```diff
@@ -16,13 +19,22 @@ export type ConfigAliasOptions = {
   defaultHidden: boolean;
 };
 
-const defaultConfig: Config = {
+export type InternalConfig = Config & {
+  /** Whether to allow RNTL to use latest and greatest improvements even if they are breaking changes. */
+  allowBreakingChanges: boolean;
+};
+
+const defaultConfig: InternalConfig = {
+  allowBreakingChanges: false,
```

Comment:
if it's an internal config, could we default to true? so that what we develop will work with the upcoming features? on the other hand, it might not work with the current features if we do that
```suggestion
  allowBreakingChanges: true,
```

Reply:
It needs to be `false`, because while it can be activated internally, it has an effect on regular users as well.

Reply:
oh true forgot that ^^'